Update wording to be ip-prefix specific. (#1232)
OctopusDeploy/docs: docs/octopus-cloud/static-ip.md

The Octopus Cloud is a multi-tenant service with several static IP addresses shared among customers in the same Azure region.

> Each Azure region uses a pool of static IP addresses. The static IP for an Octopus Cloud instance will use one from the pool and may change to another static IP from the pool under certain situations.

Reviewer: ```suggestion
Each Azure region uses a range of static IP addresses. The static IP address for an Octopus Cloud Server will be one from the range and may change to another static IP address within the range under certain situations.
```

> With a static IP address, you can lock down the ingress and egress communications between a Tentacle in your infrastructure and your Octopus Cloud instance. The built-in worker for your Octopus Cloud instance does not use an IP address from the pool of static IP addresses.

Reviewer: ```suggestion
With a static IP address, you can lock down the ingress and egress communications between a Tentacle in your infrastructure and your Octopus Cloud Server. The built-in worker for your Octopus Cloud instance does not use an IP address from the range of static IP addresses.
```

> A range of IP Addresses that your instance could be using can be found within the technical section of the instance details page. 

Reviewer: ```suggestion
The range of IP Addresses that your Octopus Cloud Server will use can be found within the technical section of the instance details page. 
```

> With a static IP address, you can lock down the ingress and egress communications between a Tentacle in your infrastructure and your Octopus Cloud Server. The built-in worker for your Octopus Cloud instance does not use an IP address from the range of static IP addresses.

Reviewer 2: ```suggestion
With a static IP address, you can lock down the ingress and egress communications between a Tentacle in your infrastructure and your Octopus Cloud Server. 

:::hint
**Note:**
The built-in worker for your Octopus Cloud Server does not use an IP address from the range of static IP addresses.
:::
```
